qemu/tests/cfg/blockdev_commit_to_nospace.cfg

```diff
@@ -0,0 +1,35 @@
+- blockdev_commit_to_nospace:
+    type = blockdev_commit_to_nospace
+    virt_test_type = qemu
+    only Linux
+    device_tag = 'data'
+    images += " ${device_tag}"
+    force_create_image_data = yes
+    remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 2G
+    image_format_data = qcow2
+    snapshot_tags_data = sn1
+
+    image_size_sn1 = 2G
+    image_format_sn1 = qcow2
+    image_name_sn1 = sn1
+
+    rebase_mode = unsafe
+    qemu_force_use_drive_expression = no
+
+    mount_dir = /mnt/test
```

Comment:
Do you think it's better to make the mnt under /tmp and give it an unusual name? e.g. mount_dir = /tmp/limited_space_mnt, because /mnt is well known for fs mounts, and we usually create sub-dirs under /mnt for fs mount purpose, and test is a common name that usually be used.